src/globals/scss/styles.scss

```diff
@@ -6,6 +6,7 @@ $css--font-face: false !default;
 $css--helpers: true !default;
 $css--body: true !default;
 $css--reset: false !default;
+$css--use-layer: true !default;
```

Comment:
Hmm, maybe I'm not following, but wouldn't this flag prevent code from being output, so it *would* belong here? 

If this flag is set to false, no layer styles are added and no CSS styles are generated. If it is set to true, all layer styles will produce CSS styles